app/models/legacy/translation.rb

```diff
@@ -0,0 +1,53 @@
+module Legacy
+
+class Translation < ActiveRecord::Base
+  establish_connection :legacy
+  include LegacyModel
+
+  def migration_data
+    new_model_name = map_model_name(remote_table)
+    new_attribute_name = map_attribute_name(new_model_name, remote_column_name)
+    puts "Translation[#{self.id}]"
+    data = {
+        # note, no need to maintain ids from legacy translation table
+        # id: self.id,
+        translatable_type: new_model_name,
+        translatable_id: remote_id,
+        translatable_attribute: new_attribute_name,
+        language_id: language,
+        text: translated_content
+    }
+    data
+  end
+
+  def migrate
+    if [1,2].include?(language)
+      data = migration_data
+      puts "#{data[:id]}: #{data[:translatable_type]}[#{data[:translatable_id]}].#{data[:translatable_attribute]}"
+      ::Translation.create(data)
+    else
+      # note, there is a lot of orphaned translation data in the current production db
+      puts "Warning, ignoring Translation[#{id}] with invalid Language reference: #{language}"
```

Comment:
Yeah, the 3s I looked at were French, but I didn't do a complete review.  It's odd that There is nothing anymore in the legacy Language table above 2.  Shall I just go and add a record in the seeds.rb for 3->French?